samples/solidity/.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
Make sure `npm run lint` still works in the base project... sometimes lint does not like finding multiple config files at different levels of the tree.